test/TwigRendererTest.php

```diff
@@ -67,14 +67,6 @@ public function assertEqualTemplatePath(TemplatePath $expected, TemplatePath $re
         }
     }
 
-    public function testShouldInjectDefaultLoaderIfProvidedEnvironmentDoesNotComposeOne()
```

Comment:
This test is removed as it was not testing any interface of the library but it was Twig test and it should be part of twig library.